resources/istio/charts/kyma/templates/destination-rule.yaml

```diff
@@ -1,7 +1,8 @@
+---
 apiVersion: networking.istio.io/v1alpha3
 kind: DestinationRule
 metadata:
-  name: istio-ingressgateway
+  name: istio-ingressgateway-dr
```

Comment:
nit: it is not needed to add suffixes with resource kind abbreviation when you fetch it you have to specify kind